### ISSUE TITLE
Box Overflow (in Add New Comment) - Wagtail Storybook

### ISSUE THREAD

**Issue by mansi0829** [wagtail/wagtail#10233]
### Issue Summary
![Screenshot from 2023-03-14 16-43-49](https://user-images.githubusercontent.com/81025790/224984559-39eb1342-47f3-4396-91b5-3b6353ed4dc5.png)

The size of the text field to add new comment is overflown

### Proposed solution

1. In /wagtail/client/src/components/CommentApp/components/Comment/index.tsx
2. Edit in css for text field required

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

-   The UI/UX for the Storybook website is not looking appropriate

**Comment by Priyansh1909** [comment]
hello, i just want to confirm that i have to create a fork make change and upload it here. this is my first contribution so i just want to confirm it.

**Comment by sohankamble** [comment]
@mansi0829  can you assign this issue to me , so I work on this.

**Comment by lb-** [comment]
@sohankamble we don't assign issues outside of the core team. 

See https://github.com/wagtail/wagtail/issues/10070 for a deep dive into your first contribution. 

Or alternatively, as this bug is unconfirmed, please see if you can reproduce it and add a screenshot. 

It looks like this is a bug in the storybook site so it may not be a huge problem, please check if this problem can exist in the actual Wagtail admin also. 

If it's a problem in Storybook only, we may not need it fixed.

**Comment by allenvtran** [comment]
So I looked at the index.tsx and saw the snippet of code that has the text "Enter your comments". This part of the code has a react component called TextArea which is comprised of many elements including <textarea>. This element is also found under main.scss for the CommentApp directory. As I checked back and forth between main.scss and .comment under style.scss, there must be some bug that changes the positioning of textarea as it should be centered within the white box, or at least no left/right padding.

A solution I found to this bug was making the padding 0 for the box. I also made the padding for the textarea to be 10px for top and bottom, 0px for the left and right. Here's what it looks like: 
<img width="1429" alt="Screen Shot 2023-03-19 at 1 19 58 PM" src="https://user-images.githubusercontent.com/112126941/226206723-bf62fce8-a881-4569-9e70-653ab86cbe86.png">

Here's the changed code in main.scss with comments:
<img width="927" alt="Screen Shot 2023-03-19 at 1 23 35 PM" src="https://user-images.githubusercontent.com/112126941/226206876-096332aa-d009-4737-96dc-5079fc15d948.png">

Let me know if this is fine and I look forward to Pull Requesting this code!

**Comment by lb-** [comment]
@allenvtran I do not think this is an issue outside of the storybook code, it's possible that some other override is making this work in the page editor that is not applied in storybook.

See screenshot below, this looks 100% ok in the actual editor.

<img width="1009" alt="Screenshot 2023-03-20 at 8 22 07 am" src="https://user-images.githubusercontent.com/1396140/226213204-aad83f51-0531-4138-9d6b-d40d9ef974cb.png">


I am inclined to close this issue as it's not really critical to Wagtail core, but if you find a way to make this look the same in the editor and the Storybook, please suggest that approach. Changing the global box styles for one edge case in the pattern library (storybook) is not going to be an appropriate solution sorry.

**Comment by allenvtran** [comment]
@lb- Where do I find that page in Storybook? Is it under the Commenting Section?

**Comment by lb-** [comment]
@allenvtran not sure without having my editor open, just search in the code for the Storybook page's title 'someone else without avatar'. That should be unique enough to get you close. 

Alternatively - look for the file that is a stories.jsx file that makes sense, there are less than 20 of them.

**Comment by allenvtran** [comment]
I can't seem to locate stories.jsx, but I do understand my mistake of changing all the comment boxes to have a zero margin.

**Comment by lb-** [comment]
I did a 3s search for the words 'Someone else' and the two files came up instantly.

* client/src/components/CommentApp/components/Comment/index.stories.tsx
* client/src/components/CommentApp/components/CommentReply/index.stories.tsx

<img width="1931" alt="Screenshot 2023-03-23 at 9 09 15 am" src="https://user-images.githubusercontent.com/1396140/227058614-dabdc57a-ce69-4124-ba8f-af4eb059aae3.png">

@allenvtran sorry I cannot help with these kinds of questions going forward. You will need to do your own digging to find out where files are in the future. Please reach out in #new-contributors for finding files if you get really stuck.

I know it can be confusing to work through new code but this is not flagged as a good first issue and is not something that we even know is worth fixing. You are free to do your own digging and propose a solution but we wont be able to help you further for 'getting started' type questions on this issue.

**Comment by SyedImtiyaz-1** [comment]
Can you please assign this issue to me.

**Comment by SyedImtiyaz-1** [comment]
@gasman So can you please tell me in which issue I can contribute

**Comment by gasman** [comment]
@SyedImtiyaz-1 This is covered by the guide @lb- linked to: #10070

**Comment by SyedImtiyaz-1** [comment]
Thanks

**Comment by pantha704** [comment]
I've submitted a fix for this issue in #13708.

The root cause was the textarea missing `box-sizing: border-box`, which caused the padding and border to add to the 100% width, making it overflow the 300px container.

Verified in Storybook - the textarea is now properly contained within the comment box.
